github/repos_rules.go

```diff
@@ -85,6 +85,21 @@ type RuleFileParameters struct {
 	RestrictedFilePaths *[]string `json:"restricted_file_paths"`
 }
 
+// RuleMaxFilePathLengthParameters represents the max_file_path_length rule parameters.
+type RuleMaxFilePathLengthParameters struct {
+	MaxFilePathLength int `json:"max_file_path_length"`
+}
+
+// RuleFileExtensionRestrictionParameters represents the file_extension_restriction rule parameters.
```

Comment:
Question: Should this possibly be?

```
// RuleFileExtensionRestrictionParameters represents the restricted_file_extensions rule parameters.
```

Reply:
Annoyingly no, the rule is called file_extension_restriction and the parameter is restricted_file_extensions